src/api/NotificationsAssistant.ts

```diff
@@ -98,22 +102,33 @@ export default class NotificationsAssistant{
 	async ClearNotifications(){
 
 		const pending = await Plugins.LocalNotifications.getPending()
-		Plugins.LocalNotifications.cancel(pending) //clear all the pending notifications
+		if(pending.notifications.length != 0){
+			Plugins.LocalNotifications.cancel(pending)
+		}
+		 //clear all the pending notifications
 		this.frequency = 0
 		this.start = 0
 		this.end = 0
 		this.verses = []		
 	}
 
+	async NoNotificationsRemaining() : Promise<Boolean>{
+		const pending = await Plugins.LocalNotifications.getPending()
+		console.log(pending)
+		console.log(pending.notifications.length)
+		console.log(pending.notifications.length == 0)
+		return pending.notifications.length == 0
```

Comment:
Again, == is bad style. Check for typescript compiler warnings, which should show up in your terminal on running npm start. == should show up as a warning with a suggestion to change to ===